packages/flutter_tools/lib/src/web/compile.dart

```diff
@@ -251,28 +251,28 @@ enum WebRendererMode {
 /// The correct precompiled artifact to use for each build and render mode for DDC with AMD modules.
 // TODO(markzipan): delete this when DDC's AMD module system is deprecated, https://github.com/flutter/flutter/issues/142060.
 const Map<WebRendererMode, HostArtifact> kAmdDartSdkJsArtifactMap = <WebRendererMode, HostArtifact>{
-  WebRendererMode.auto: HostArtifact.webPrecompiledAmdCanvaskitAndHtmlSoundSdk,
-  WebRendererMode.canvaskit: HostArtifact.webPrecompiledAmdCanvaskitSoundSdk,
-  WebRendererMode.html: HostArtifact.webPrecompiledAmdSoundSdk,
+  WebRendererMode.auto: HostArtifact.webPrecompiledAmdCanvaskitAndHtmlSdk,
+  WebRendererMode.canvaskit: HostArtifact.webPrecompiledAmdCanvaskitSdk,
+  WebRendererMode.html: HostArtifact.webPrecompiledAmdSdk,
 };
 
 /// The correct source map artifact to use for each build and render mode for DDC with AMD modules.
 // TODO(markzipan): delete this when DDC's AMD module system is deprecated, https://github.com/flutter/flutter/issues/142060.
 const Map<WebRendererMode, HostArtifact> kAmdDartSdkJsMapArtifactMap =
     <WebRendererMode, HostArtifact>{
-      WebRendererMode.auto: HostArtifact.webPrecompiledAmdCanvaskitAndHtmlSoundSdkSourcemaps,
-      WebRendererMode.canvaskit: HostArtifact.webPrecompiledAmdCanvaskitSoundSdkSourcemaps,
-      WebRendererMode.html: HostArtifact.webPrecompiledAmdSoundSdkSourcemaps,
+      WebRendererMode.auto: HostArtifact.webPrecompiledAmdCanvaskitAndHtmlSdkSourcemaps,
+      WebRendererMode.canvaskit: HostArtifact.webPrecompiledAmdCanvaskitSdkSourcemaps,
+      WebRendererMode.html: HostArtifact.webPrecompiledAmdSdkSourcemaps,
     };
 
 /// The correct precompiled artifact to use for each build and render mode for
 /// DDC with DDC library bundle module format. Only artifacts with sound
 /// null-safety are provided.
```

Comment:
nit: I think we can just remove this qualifier here and below now.

Reply:
Done!